src/documentation/pages/components/icon/examples/_base.vue

```diff
@@ -0,0 +1,44 @@
+<template lang="pug">
+<ComponentExample title="Base" id="base" padding="-p--0" :tabs="exampleTabs">
+  .-p--3(slot="example")
```

Comment:
Can we use this?

```suggestion
  <Wrapper slot="example">
```

Reply:
@josecarlosgonzalezv As per my conversation with @evansito I think Wrapper is not needed for this specific scenario. I implemented it in the same way for header component as well.